### PR TITLE
Set the baudrate to default and reduce prints in blinky

### PR DESCRIPTION
**Pull request by deepikabhavnani**
Issue: https://github.com/ARMmbed/mbed-os-example-blinky/issues/153

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -8,17 +8,25 @@
...
+    int count = 0;
     while (true) {
         // Blink LED and wait 0.5 seconds
         led1 = !led1;
-        wait(0.5f);
+        wait_ms(SLEEP_TIME);
 
-        // Following the main thread wait, report on the current system status
-        sys_state.report_state();
+        if ((0 == count) || (PRINT_AFTER_N_LOOPS == count)) {
```
Comment (kegilbert) [inline main.cpp]:
```suggestion
        if ((count == 0) || (count == PRINT_AFTER_N_LOOPS)) {
```

**Comment by kegilbert** [comment]
LGTM besides a minor style nit. This will cause a minor blip in the blinking rate every 10 seconds which shouldn't be a problem (I haven't run the new changes yet, not sure on how noticeable it will be).

Before this is merged we need a PR up to update the documentation as well.

**Comment by 0xc0170** [comment]
> Before this is merged we need a PR up to update the documentation as well.

in the docs or here in readme?

File: mbed_app.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
     "target_overrides": {
         "*": {
-            "platform.stdio-baud-rate": 115200,
```
Comment (0xc0170) [inline mbed_app.json]: 👍

File: main.cpp (hunk 1 above)
Comment (deepikabhavnani) [reply main.cpp]: Will stick to constants on left side of comparison :-)

**Comment by deepikabhavnani** [comment]
> Before this is merged we need a PR up to update the documentation as well.
> in the docs or here in readme?

Updated the Readme. @kegilbert - Let me know if anything else should be updated

**Comment by kegilbert** [comment]
https://github.com/ARMmbed/mbed-os-5-docs/pull/891 Put up, this PR should be good to go.

**Comment by deepikabhavnani** [comment]
> ARMmbed/mbed-os-5-docs#891 Put up, this PR is good to go on my end.

**Pull request merged**